### ISSUE TITLE
Liste des flux RSS difficiles à afficher sur smartphone

### ISSUE THREAD

**Issue by B4rb3rouss** [BlogoText/blogotext#351]
La liste des flux RSS suivis est franchement chiante à afficher sur smartphone.
Un peu de CSS et ça va mieux.

**Comment by remrem** [comment]
tu peux revoir ton PR ?

**Comment by B4rb3rouss** [comment]
Oui. Bizarre, en local travis ne me trouvait pas de problèmes. Souci avec git, j'étais avec l'upstream https://github.com/BoboTiG/blogotext.git.

Désolé. Normalement c'est réglé.
